apps/etterlatte-behandling/src/test/kotlin/VerdikjedeTest.kt

```diff
@@ -371,6 +393,77 @@ class VerdikjedeTest : BehandlingIntegrationTest() {
                 assertEquals(BehandlingStatus.IVERKSATT, behandling.status)
             }
 
+            // KLAGE
+            // Sniker seg inn her siden vi trenger en sak med en iverksatt behandling
```

Comment:
sneaky

Reply:
Lurer på om det er mer smud om denne får sin egen løype da dette er en egen verdikjede

Reply:
Kanskje, men det blir duplisering av hele kjeden fram til linje 394 for å få en iverksatt behandling på saken :/

Reply:
Ja nei worth a shot mby

Reply:
Jeg tar ikke inn den dupliseringen nå i alle fall. Jeg tenker gevinsten i å ha det i to forskjellig integrasjonstester går helt bort når man uansett må vedlikeholde begge hvis noe skjer med verdikjeden i opprettelse av behandling -- så de blir strengt tatt ikke uavhengige tester